ush/python/pygfs/task/aero_emissions.py

```diff
@@ -37,27 +44,50 @@ def __init__(self, config: Dict[str, Any]) -> None:
         """
         super().__init__(config)
 
-        local_variable = "something"
+        current_datetime = add_to_datetime(
+            self.task_config.current_cycle,
+            to_timedelta(f"{self.task_config.STARTHOUR}H"),
+        )
+        nforecast_hours = self.task_config["FHMAX_GFS"]
+        nforecast_days = nforecast_hours // 24 + 1
+        forecast_dates = [current_datetime + timedelta(days=i) for i in range(nforecast_days)]
 
         localdict = AttrDict(
-            {'variable_used_repeatedly': local_variable}
+            {"cdate": current_datetime, "nforecast_days": nforecast_days, "forecast_dates": forecast_dates}
         )
 
         # Extend task_config with localdict
         self.task_config = AttrDict(**self.task_config, **localdict)
 
+        # Read the aero_emission.yaml file for common configuration
+        logger.info(
+            f"Read the prep_emission configuration yaml file {self.task_config.PREP_EMISSION_CONFIG}"
+        )
+        self.task_config.aero_emission_yaml = parse_j2yaml(
+            self.task_config.PREP_EMISSION_CONFIG, self.task_config
+        )
+        logger.debug(
+            f"aero_emission_yaml:\n{pformat(self.task_config.aero_emission_yaml)}"
+        )
+
     @staticmethod
     @logit(logger)
-    def initialize() -> None:
-        """Initialize the work directory
-        """
+    def initialize(aero_emission_yaml: Dict) -> None:
+        """Initialize the work directory"""
+        logger.info("Copy Static Data to run directory")
+        # FileHandler(aero_emission_yaml.aero_emissions.fix_data).sync()
```

Comment:
Is this still a `TODO`?

Reply:
The data is copied in the configure step here.  Removing comment